build.gradle

```diff
@@ -21,10 +21,7 @@ android {
 
     defaultConfig {
         targetSdkVersion eclipseProperties.getProperty('target')-'android-'
-
-        // note that this code assumes it's building the latest tag, and that all tags are present in the repository;
-        // older tags can be built by cloning only a subsection of the history
-        versionCode 'git tag'.execute(null, projectDir) .text.readLines().size()
+        versionCode 'git rev-list --count HEAD'.execute(null, projectDir) .text.readLines().size()
```

Comment:
`.readLines().size()` won't work here, will it? The output of that git command is just a number.

Reply:
You're right!  Derp!  Why did I think I had that working?

Reply:
Would you be okay with an amended commit for this?  Was basically just a fat finger error on my part.

Reply:
That's fine!

Reply:
Alright, should be good now.